### ISSUE TITLE
OC patches

### ISSUE THREAD

**Issue by pulpoff** [ophub/kernel#1]
why did you not include my overclock patches?
see the difference ..
https://browser.geekbench.com/v5/cpu/compare/11666486?baseline=11667692
![Screenshot from 2021-12-18 00-50-37](https://user-images.githubusercontent.com/40654959/146620380-ab2950c4-a3cd-4070-8973-5684ddcdc722.jpg)

**Comment by pulpoff** [comment]
linux-5.15.y.tar.gz uploaded

**Comment by ophub** [comment]
Hello, the files in the current kernel library are compiled by flippy dumped. I sent him your patch.

**Comment by ophub** [comment]
You are familiar with these patches. If it is convenient, it is recommended that you directly submit the patches to his kernel source code so that he can directly compile them in the future.

https://github.com/unifreq/linux-5.15.y

**Comment by ophub** [comment]
Flippy reported that pressurized overclocking is not applicable to all models and may cause instability, so it is not added to the unified core file. You can use the kernel compilation script to compile separately.

**Comment by pulpoff** [comment]
> You shared [display resolution 2560x1440](https://github.com/tobetter/linux/commit/248da29964751908c36e2c5558385ec09aed87ad) has been added.
Thanks a lot!

I have attached my meson-g12b-s922x.dts file, this overclocks a53 to 2.2Ghz and A75 to 2.6Ghz, runs stable on my platform for at least a week with heavy kernel compile loads and gnome desktop same time.

File also has 2.62 and 2.63 Ghz frequencies added, switching them on with armbian-config causes issues like Geekbench crashes and some random freezes after a couple of days, so 2.6 is maximum I guess.
I have also tried up to 2.7Ghz, the highest clock that box runs is 2.68, but has many issues. 2.64 is highest "testable" clock, and probably would work with active cooling.
